service/routes.py

```diff
@@ -87,12 +80,25 @@ def update_in_stock(inventory_item_id):
     return make_response(jsonify(inventory_item.serialize()), status.HTTP_200_OK)
 
 
+@app.route("/inventories/<int:inventory_item_id>", methods=["DELETE"])
+def delete_inventory_items(inventory_item_id):
+    """
+    Delete an inventory item
+
+    This endpoint will delete a InventoryItem based the id specified in the path
+    """
+    app.logger.info("Request to delete inventory item with id: %s", inventory_item_id)
+    inventory_item_id = InventoryItem.find(inventory_item_id)
+    if inventory_item_id:
+        inventory_item_id.delete()
+
+    app.logger.info("Finished deleting inventory item [%s]", inventory_item_id)
+    return make_response("", status.HTTP_204_NO_CONTENT)
+
+
 ######################################################################
 #  U T I L I T Y   F U N C T I O N S
 ######################################################################
-"""
-This method is copied straight out of the lab. Please modify if needed.
```

Comment:
`pylint` was complaining about this as well, and I think it's fine to remove it. If we feel strongly that it should still be here, I think we just convert it to a normal comment (rather than a block comment) to remove the warning.